gf-ui-lib/src/components/Map/Map.fixtures.ts

```diff
@@ -8,94 +9,101 @@ export const markers = [
   {
     id: 1,
     name: "Golden fleece",
-    x: 1329,
-    y: 230,
+    x: 42.24625,
+    y: 42.68017,
```

Comment:
It's a magic number. Please create a consts with a good namig. Check othe strings in this file

Reply:
Done